common/util.go

```diff
@@ -404,3 +404,47 @@ func uniqueVolumeMounts(mounts []v1.VolumeMount) []v1.VolumeMount {
 	}
 	return rMounts
 }
+
+// AreSlicesEqual returns true if the two provided string slices are equal.
+func AreSlicesEqual(first []string, second []string) bool {
+	if len(first) == 0 && len(second) == 0 {
```

Comment:
Can we not sort and apply reflect.DeepEqual?

Reply:
No, the slices must be sorted if we want to apply `reflect.DeepEqual`. Also, as far as I recall `reflect.DeepEqual` is much slower than a loop in this case.